build/lin.sh

```diff
@@ -124,6 +124,7 @@ VERSION_PANGO=1.49.3
 VERSION_SVG=2.52.4
 VERSION_AOM=3.2.0
 VERSION_HEIF=1.12.0
+VERSION_CGIF=0.0.2
```

Comment:
This is currently missing in `versions.json`.

Reply:
Added, thanks.